tasks/move-granules/tests/test-index.js

```diff
@@ -253,24 +243,6 @@ test.serial('Should not overwrite CMR file type if already explicitly set', asyn
   t.is('userSetType', cmrFile[0].type);
 });
 
-test.serial('Should preserve object tags.', async (t) => {
```

Comment:
re: comment above - might these be worth skipping, or rather leaving a skipped stub with a comment referencing the integration test so someone doesn't try to re-write if they're looking closely at code coverage, etc?    Mocking it to just test the flag, or ... ?

Reply:
The `moveGranules` function has a bunch of different possible paths through the code, which results in the actual `S3.moveObject` call being made in a few different places (and even different files). This test was only testing one of those paths. I agree that we should fix this, but I think it's going to require refactoring that's beyond the scope of this ticket. I can bring it up in the next architecture meeting.

Reply:
That seems fine.  We could probably also write a ticket to track the concern as well, but I'm not stuck on that.